composer.json

```diff
@@ -1,12 +1,20 @@
 {
     "name": "atoum/standard-edition",
+    "type": "package",
+    "keywords": ["TDD","atoum","test","unit testing","atoum-edition"],
+    "homepage": "http://www.atoum.org",
     "require": {
         "atoum/atoum": "^2.9",
         "atoum/stubs": "^2.5",
         "atoum/visibility-extension": "dev-self_register as 1.1",
         "atoum/config-extension": "dev-merge_prs as 1.0",
         "atoum/reports-extension": "dev-self_register as 2.9"
     },
+    "authors": [
```

Comment:
does adding authors will limit contributions ? plus this a a team repo, it's not only mine. should we keep that ? (also, if we keep it, the real name si prefered in the composer.json : https://getcomposer.org/doc/04-schema.md#authors )